ui-components/src/locales/general.json

```diff
@@ -1063,7 +1064,7 @@
       "maxSize": "Maximum Waitlist Size",
       "maxSizeQuestion": "What is the maximum size of the waitlist?",
       "open": "Open Waitlist",
-      "openQuestion": "Is the waitlist open?",
```

Comment:
Just a reminder that if we are changing a string we need to remove the keys in the translations files so that we know the string needs to be updated / is now wrong (unless maybe this is just a partners string and then NBD!)

Reply:
Good point…I searched around and this is only for partners, so we I think we're in the clear.